src/models/orderListModel.ts

```diff
@@ -0,0 +1,17 @@
+import {extendObservable} from 'mobx';
+
+class OrderListModel {
```

Comment:
hmm, but this is an `Order`

Reply:
btw it's defined in models somewhere, probably it's a good idea to reuse/extend existing

Reply:
It contains a different fields